diagnostics/diagnostics.go

```diff
@@ -27,28 +27,30 @@ import (
 	"sync"
 	"time"
 
+	"github.com/pilosa/pilosa"
 	"github.com/shirou/gopsutil/host"
 	"github.com/shirou/gopsutil/mem"
 	"github.com/sony/gobreaker"
 )
 
+// Ensure ActiveDiagnostics implements interface.
+var _ pilosa.Diagnostics = &ActiveDiagnostics{}
```

Comment:
Apply this check to the NopDiagnostics too.

Reply:
applied in 2f74fd4aad3a4e247d166bdd76b6181662be5443